compiler/noirc_evaluator/src/ssa/opt/inlining.rs

```diff
@@ -19,6 +19,7 @@ use crate::ssa::{
     ssa_gen::Ssa,
 };
 use fxhash::FxHashMap as HashMap;
+use im::HashSet as ImmutableHashSet;
```

Comment:
```suggestion
```
Might as well refer to this as `im::HashSet` - it is shorter and avoids having different names for the same thing across the codebase.